nodejs/week1/src/backend/routes/cheap-meals.js

```diff
@@ -0,0 +1,18 @@
+const mealArray = require("../data/meals.json");
+const reviewsArray = require("../data/reviews.json");
+
+const cheapMeal = function(request, response) {
+  const mealResault = mealArray.map(mealItem => {
+    if (mealItem.price <= 50) {
+      mealItem.reviews = reviewsArray.filter(
+        reviewsItem => mealItem.id === reviewsItem.mealId
+      );
+      return mealItem;
+    } else {
+      return;
```

Comment:
If you check the result of this route you would see that for the not cheap meals, there will be a `null` in the result. We don't want that so you should try to change the logic here a bit, maybe to something like this:
- First find the cheap meals
- And then do a map to add the reviews

Reply:
The same issue applies to the large meals route.